### ISSUE TITLE
multiple JSON-values into the same property not working

### ISSUE THREAD

**Issue by kaps727** [dhlab-basel/json2typescript#88]
I am following below section of documentation

> Tip: It is possible to deserialize multiple JSON-values into the same property. This can be useful if you have multiple JSON sources with inconsistent names, but want a single target object. In the process of serialization, the first value will be used.
> 
>     @JsonProperty("jsonPropName", String, true)
>     @JsonProperty("jsonPropertyName", String, true)
>     name: string = undefined;

but its not giving me expected output.

NOTE: All other properties are getting converted as expected with only one @JsonProperty decorator.

here is link of reproduction of this issue.

https://stackblitz.com/edit/angular-uh3d4a

**Comment by andreas-aeschlimann** [comment]
This is a future feature that has already been added to the code on GitHub. I shouldn't have pushed the recent commit to the master branch.

Please always refer to the documentation on npmjs website, because that's the documentation matching the currently released version!

The ReadMe on this GitHub already implements this new feature and after doing some more integration tests I will release it (probably within 7 days).

TLDR: As of now, you can only use 1 decorator for the property.

**Comment by andreas-aeschlimann** [comment]
I released the current version v1.1.1 now.

Unfortunately, I decided to drop support for multiple decorators just for now. One of the reason is that the logic was not consistent in my eyes. I have to rethink the idea and discuss with my colleagues what will be best for the future.

**Comment by kaps727** [comment]
Okay. Waiting for the same.

**Comment by tlmurphy** [comment]
That piece of documentation still lives in the readme as well as on npmjs. It should probably be removed as it's a little misleading since it doesn't work properly as of yet.

**Comment by andreas-aeschlimann** [comment]
That is not good, can you show us where? Just on NPM or also GitHub?

**Comment by tlmurphy** [comment]
Right above the https://github.com/dhlab-basel/json2typescript/blob/master/README.md#first-parameter-classidentifier-optional section.

```
Tip: Make sure you import JsonObject from json2typescript.

    @JsonProperty("jsonPropName", String, true)
    @JsonProperty("jsonPropertyName", String, true)
    name: string = undefined;
```

It's in the same place on npm as well.

**Comment by andreas-aeschlimann** [comment]
I don’t even understand what it is doing there, it’s in the wrong chapter. Will be deleted soon.

**Comment by cristianmiranda** [comment]
Hi! Any updates on this one? 🙄
